java/ql/test/library-tests/frameworks/android/content-provider/test.ql

```diff
@@ -0,0 +1,13 @@
+import java
+import semmle.code.java.dataflow.FlowSources
+import TestUtilities.InlineFlowTest
+
+class ProviderTaintFlowConf extends DefaultTaintFlowConf {
+  override predicate isSource(DataFlow::Node n) { n instanceof RemoteFlowSource }
+}
+
+class ProviderInlineFlowTest extends InlineFlowTest {
+  override DataFlow::Configuration getValueFlowConfig() { none() }
+
+  override DataFlow::Configuration getTaintFlowConfig() { result instanceof ProviderTaintFlowConf }
```

Comment:
`ProviderTaintFlowConf` already overrides the `DefaultTaintFlowConf`, so I think this override is superfluous.

Reply:
True, fixed in d5f675c. Thanks!